low-level/sspl_ll_d

```diff
@@ -815,6 +817,16 @@ if __name__ == "__main__":
 
         #DO NOT EDIT: Marker comment to dynamically add code to start the code coverage scope
 
+        # Get server fru list
+        executor = IpmiFactory().get_implementor("ipmitool")
+        server_fru_list = executor.get_server_fru_list()
+
+        if os.path.exists(SERVER_FRU_LIST_FILE):
```

Comment:
why to persist? Everytime sspl bootup, can build this list ?

Reply:
Ack.